### PR TITLE
Update Working_with_trend_predictions.md

### PR DESCRIPTION
**Pull request by VeerleLedoux**
Some incorrect information was fixed. For instance, the old version mentioned "trend predictions can be disabled in the user settings", this is not true. Reference was added to Proactive cap detection, a strongly related feature which has a dependency on the Trend Prediction feature. Clarification was added on the different prediction types: users often wonder what these represent or what the difference is between the different types. Clarification was added on why some metrics might not have a prediction.

### CODE REVIEW

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md [hunk 1]
```diff
@@ -28,41 +27,63 @@ The predicted trend is displayed as a line surrounded by three shaded areas:
 
 > [!NOTE]
 >
-> - The probability percentages are based on the assumptions made at the moment the graph is generated. If the behavior of the data changes afterwards, the probability may need to be adjusted.
-> - If a recurring pattern is found, the predicted trend is based on this pattern. Otherwise, the predicted trend should be considered as the average future value.
+> - The predicted trend is generated by a machine learning model fit on the historical trend data of the metric.
```
Comment (EdithLansens) [inline user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md:30]:
```suggestion
> - The predicted trend is generated by a machine learning model trained on the historical trend data of the metric.
```

I am unsure about this wording of a model "fit" on trend data. Would "trained" be more accurate?

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md [hunk 2]
```diff
@@ -28,41 +27,63 @@ The predicted trend is displayed as a line surrounded by three shaded areas:
...
-- Long-term prediction is displayed with slightly longer dashes
+  A short-time prediction (prior to DataMiner 10.0.12: mid-term prediction) is generated by a model fit on the 5-minute averaged trend data. The use of 5-minute averages usually over extended time spans means that the forecast time horizon for a short-term prediction is typically longer compared to a high-precision prediction.
+
+- **Mid-term prediction** is displayed with long dashes.
+
+  A mid-term prediction is generated by a model fit on the 1-hour averaged trend data. This type of trend prediction offers a forecast time horizon that is typically longer compared to a short-term prediction.
+
+- **Long-term prediction** is displayed with slightly longer dashes.
+
+  A long-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to the future.
```
Comment (EdithLansens) [inline user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md:51]:
```suggestion
  A long-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to make forecasts for the future.
```

Is this different wording correct @VeerleLedoux ?

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md [hunk 3]
```diff
@@ -28,41 +27,63 @@ The predicted trend is displayed as a line surrounded by three shaded areas:
...
-- Mid-term prediction is displayed with short dashes.
+  A mid-term prediction is generated by a model fit on the 5-minute averaged trend data. The use of 5-minute averages usually over extended time spans means that the forecast time horizon for a mid-term prediction is typically longer compared to a short-term prediction.
 
-- Long-term prediction is displayed with long dashes.
+- **Long-term prediction** is displayed with long dashes.
 
-- Full-term prediction, which is based on either daily data points or a server-side aggregated set of one-hour data points, is also displayed with long dashes. (Available from DataMiner 10.0.11 onwards.)
+  A long-term prediction is generated by a model fit on the 1-hour averaged trend data. This type of trend prediction offers a forecast time horizon that is typically longer compared to a mid-term prediction.
+
+- **Full-term prediction** is also displayed with long dashes (Available from DataMiner 10.0.11 onwards).
```
Comment (EdithLansens) [inline user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md:67]:
```suggestion
- **Full-term prediction** is also displayed with long dashes (Available from DataMiner 10.0.11 onwards).
  A full-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to the future.
```
Am I correct to assume that the description you provided for a long-term prediction (from 10.0.12 onwards) above can also be entered here?

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md (hunk 1 above)
Comment (VeerleLedoux) [reply user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md:30]: I am ok with changing it to "training"

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md (hunk 2 above)
Comment (VeerleLedoux) [reply user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md]: I am not sure your wording expresses exactly what I meant.  I mainly wanted to make clear that the long-term prediction will just show the global long-term trend line in the data and will not contain the more short-term fluctuactions etc

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md (hunk 3 above)
Comment (VeerleLedoux) [reply user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md:67]: yes, the "long-term prediction" of versions later than 10.0.12, used to be called "full-term prediction" in older versions. The description can be copied

**Pull request merged**